conf/wmi.conf.example

```diff
@@ -64,13 +65,49 @@ value = explorer.exe
 action = allow
 EOT
 
+# Detect if FortiClient.exe is running, if yes the Security Event is triggered
+[Forticlient-Enable]
+request=SELECT Name FROM Win32_Process
+namespace=ROOT\cimv2
+action= <<EOT
+[ActiveProcess]
+attribute = Name
+operator = match
+value = FortiClient.exe
+
+[1:ActiveProcess]
+action=trigger_security_event
+action_param = mac = $mac, tid = PROCESS_FORTI, type = CUSTOM
+EOT
+on_tab=0
+
+
 # Detect if the SCCM client is installed
 [SCCM]
 request=select * from Win32_Process where name='CcmExec.exe'
 action=[sccm]
 namespace=ROOT\cimv2
 on_tab=1
 
+# Detect if the SCCM client is installed and if yes an Security Event is triggered
```

Comment:
Same as above.

For running process that shouldn't be running, create a meaningful example, like someone running a BitTorrent client, or a Tor client, etc.